charts/crowdsec/values.schema.json

```diff
@@ -26,13 +26,50 @@
                     "items": {
                         "$ref": "#/definitions/Acquisition"
                     }
+                },
+                "manualAcquisition": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ManualAcquisition"
+                    }
                 }
             },
             "required": [
                 "acquisition"
             ],
             "title": "Agent"
         },
+        "ManualAcquisition": {
```

Comment:
For the validation, I think it's more complex than that, if you take a look at the [available datasources](https://docs.crowdsec.net/docs/next/data_sources/intro). Each one has it own parameters.

Maybe ignoring this part for now, or you can build the complex schema following condition on the `source`